interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: Calling super in a class 'extracts' the initialize method that is within the inherited class. It grants access to the symbols and values that were already defined in the parent class without having the define them again in the child class. An example would be there is a class CocaCola with a def initialize, and within that initialize there is a symbol of @flavor. Then the class DietCoke < CocaCola could call super to have access to @flavor, without remaking the symbol within class DietCoke.
 
-Researched answer:
+Researched answer: Calling super is used to reference the superclass of the current method. It references the initialize method within the parent class, and acts ad if you were calling the the original method. An example would be there is a class CocaCola with a def initialize, and within that initialize there is a symbol of @flavor. Then the class DietCoke < CocaCola could call super to have access to @flavor, without remaking the symbol within class DietCoke.
 
 2. What is a gem?
 
-Your answer:
+Your answer: A gem is a tool used to install dependencies and packages containing pre-built data. 
 
-Researched answer:
+Researched answer: A gem is a library of data that has specific purposes. It can also contain files or dependencies required or related to that purpose. Gems are open source, which allows developers to have access to programs and data other developers have created.
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: No idea
 
-Researched answer:
+Researched answer: A relational database is a type of database that structures data in the form of a table with rows and columns, such as PostgreSQL. There are many different types of databases. To name a few, there are centralized, cloud, graph, object-oriented databases, among many others.
```

Comment:
More content can be a brief explanation of the examples you provided and adding a few companies that use each